Code/src/main/resources/ui/SimulateBlock.fxml

```diff
@@ -0,0 +1,27 @@
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.*?>
+<?import nl.utwente.viskell.ui.components.SimulateBlock?>
+<?import javafx.scene.control.Button?>
+<fx:root type="nl.utwente.viskell.ui.components.SimulateBlock" xmlns:fx="http://javafx.com/fxml/">
+    <BorderPane styleClass="display, block">
+        <center>
+            <VBox alignment="CENTER">
+                <!-- silly extra HBox to not break the argumentSpace css -->
```

Comment:
Maybe someone knowing about css could fix this instead of making more copies of a workaround.